src/features/unified-share-modal/__tests__/SharedLinkSection.test.js

```diff
@@ -185,4 +185,17 @@ describe('features/unified-share-modal/SharedLinkSection', () => {
             expect(wrapper).toMatchSnapshot();
         });
     });
+
+    test('should render disabled create share link message', () => {
```

Comment:
Minor: these tests should all say "shared" link

Reply:
👍